tests/src/test/java/org/wildfly/plugins/bootablejar/maven/goals/AbstractDevWatchTestCase.java

```diff
@@ -94,7 +94,11 @@ public void run() {
                     } else {
                         cmd.add("mvn");
                     }
-                    String[] mvnCmd = {"-f", pomFile.toAbsolutePath().toString(), "wildfly-jar:dev-watch", "-e", prop};
+                    String localRepository = System.getProperty("maven.repo.local");
+                    String[] mvnCmd = {"-f", pomFile.toAbsolutePath().toString(), "wildfly-jar:dev-watch", "-e", prop,
+                            // need to use the same repository as it is used for build
+                            localRepository != null && !localRepository.isEmpty() ? "-Dmaven.repo.local=" + System.getProperty("maven.repo.local") : ""
```

Comment:
You could use localRepository instead of calling: System.getProperty("maven.repo.local")